torchvision/prototype/transforms/functional/_color.py

```diff
@@ -226,19 +226,15 @@ def adjust_hue_image_tensor(image: torch.Tensor, hue_factor: float) -> torch.Ten
         return image
 
     orig_dtype = image.dtype
-    if image.dtype == torch.uint8:
-        image = image / 255.0
```

Comment:
Instead of doing the conversion manually, I've opted to use our kernel for this. Note that this also implicitly converts to `float32` since the divisor is a `float`.